mission_control/navigator_launch/launch/subsystems/tf.launch

```diff
@@ -1,18 +1,18 @@
 <launch>
-    <node pkg="tf" type="static_transform_publisher" name="measure_point_tf" args="1.2319 0 0  0 0 0 1  /base_link /measurement 100" /> <!-- Very front center of top part -->
+    <node pkg="tf" type="static_transform_publisher" name="measure_point_tf" args="1.2319 0 0  0 0 0 1  /base_link /measurement 10" /> <!-- Very front center of top part -->
     <node pkg="tf" type="static_transform_publisher" name="ins_tf" args="-0.652272 0.136525 0.192278  0 0 0 1  /measurement /ins 10" />
     
     <!-- LIDAR -->
-    <node pkg="tf" type="static_transform_publisher" name="lidar_tf" args="-0.277622 0 0.648208 0 0 0 1  /measurement /velodyne 100" />
-    <node pkg="tf" type="static_transform_publisher" name="sick_tf" args="0.5334 -0.0254 -0.6858  0 0 0 1  /measurement /sick 100" />
+    <node pkg="tf" type="static_transform_publisher" name="lidar_tf" args="-0.277622 0 0.648208 0 0 0 1  /measurement /velodyne 10" />
+    <node pkg="tf" type="static_transform_publisher" name="sick_tf" args="0.5334 -0.0254 -0.6858  0 0 0 1  /measurement /sick 10" />
 
     <!-- Cameras -->
-    <node pkg="tf" type="static_transform_publisher" name="stereo_tf" args="0.017272 -0.100076 0.19812 0.56679 -0.56679 0.42278 -0.42278 /measurement /stereo_front 100" />
```

Comment:
So we're not going to have a TF for the stero frame? Also, if the right cam tf is great, why don't we just defined the left TF in reference to the right tf. We should know this transformation pretty accurately (a precise measurement of the y direction distance between them).

So something like:
```
<node pkg="tf" type="static_transform_publisher" name="stereo_left_cam_tf" args="-0.200152 0 0  0 0 0 1 / stereo_right_cam  /stereo_left_cam 100" />
```